projects/smoke_drv120/src/main.c

```diff
@@ -0,0 +1,57 @@
+#include <stdbool.h>
+
+#include "delay.h"
+#include "drv120_relay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
```

Comment:
You should have a comment here telling the hardware person running the smoke test what it does, i.e. it periodically opens and closes the relay and logs any error conditions that the DRV120 signals on the status pin.